src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(incomplete_features)]
```

Comment:
Is this needed for something? After we remove this I can merge!

Reply:
Ok I removed that line (but now get a clippy warning)

Reply:
Thanks! I think that's okay, should address that in a separate PR if want to fix

Reply:
what clippy config are you using? or are you getting this error too?

Reply:
I get it when I do +nightly, since generic_const_exprs is incomplete, I'd rather keep that warning on. If you're getting the warning on stable then yeah I must be using different version.

```
> cargo clippy --version
clippy 0.1.62 (a8314ef7 2022-06-27)
```

Reply:
ok, i’ll check that later